api/ruby/enterprise/change-domains-in-links.rb

```diff
@@ -20,12 +22,22 @@
   kit.auto_paginate = true
 end
 
-unless ARGV.length == 2
+unless ARGV.length >= 2
   puts "Specify domain names to change using the following format:"
   puts "- change-domains.rb old_domain new_domain"
   exit 1
 end
 
+options = OpenStruct.new
+options.noop = false
+
+OptionParser.new do |parser|
+  parser.on("-n", "--noop", "Find the links, but don't update the content. \n\t\t\t\t\s\s\s\s\sPipe this to a CSV file for a report of all links that will be changed.") do |v|
+    options.noop = v
+  end
+end.parse!
```

Comment:
Pretty sure you can pass `Pipe this to a CSV file for a report of all links that will be changed.` as a new string argument to get the desired output.

```ruby
parser.on(
  "-n", 
  "--noop", 
  "Find the links, but don't update the content.",
  "Pipe this to a CSV file for a report of all links that will be changed."
) do |v|
```